install/kubernetes/Makefile

```diff
@@ -44,7 +44,7 @@ update-chart: $(ROOT_DIR)/VERSION # Update chart versions to point to the curren
 	$(ECHO_GEN)cilium/Chart.yaml
 	$(QUIET)grep -lR -e 'version:' -e 'appVersion:' $(CILIUM_CHARTS)/ \
 		| xargs -L 1 $(SED) -i -e 's/'$(CILIUM_CHART_REGEX)'/\1 '$(VERSION)'/g'
-	$(QUIET)$(SED) -i 's;icon:.*;icon: $(LOGO_BASE_URL)/cilium@$(CILIUM_BRANCH)/$(LOGO_PATH);' $(CHART_FILE)
+	# $(QUIET)$(SED) -i 's;icon:.*;icon: $(LOGO_BASE_URL)/cilium@$(CILIUM_BRANCH)/$(LOGO_PATH);' $(CHART_FILE)
```

Comment:
Why was this change made? It doesn't seem to be related to the goal of the PR.

Reply:
@gandro @youngnick @aditighag I know this is a nit, but how is this falling through review?

Reply:
It was probably an oversight. Based on my update, I only looked at the files for my CODEOWNERS (contributing), so I most likely didn't look at the changes in this file. 

`/install/kubernetes/ @cilium/sig-k8s @cilium/helm`

Reply:
That was not intentional. I had to fight a bit to get the makefiles to work on my computer. I eventually did. I don't remember this change specifically, but I probably commented it out for testing and didn't catch that I committed it. I can open a PR to fix that if you'd like.

Reply:
I submitted #28929. The checks are passing and I believe I did the "fixes" correctly.

Reply:
Thanks all :+1: My goal with starting the thread was to understand whether there's some underlying issue on some platforms that might have motivated this change in the first place (in which case maybe we can tweak the makefiles/scripts to be more platform-friendly?) and to get a better understanding of why unintentional changes may be introduced into the tree.

While in general I think we have a policy not to add commented code to the tree, it is ultimately up to the reviewers to enforce these constraints and I'd typically expect reviewers to collectively lay eyes on every line of code and have an understanding about how that code fits into the broader picture. In this case it seems like the submission slipped through both Brad's filter as he submitted the PR and the reviewer's filters as they accepted the PR. 

There's always a small chance of this sort of thing happening, and I understand that contributors can make mistakes or have a slip of oversight, particularly when changes are introduced later in the review cycle. Given the brief retrospective we've collectively done here, I don't think there is a systematic issue to be concerned about beyond this PR; it's just a classic case of not realizing that there was another change that came in after initial review which we would usually not accept. Fortunately the consequences in this case are low.

I hope that this discussion has helped with sharing understandings and expectations around the PR submission process going forward. I appreciate the quick feedback & solution @bradwhitfield :pray: